sporkify/frontend/templates/inventory.html

```diff
@@ -49,7 +50,7 @@
             </li>
             <li id="bullet4_purchased">
                 <label id="purVendor">Vendor<br>
-                    <select id="vendor" name="vendor">
+                    <select id="endor" name="vendor">
```

Comment:
![image](https://user-images.githubusercontent.com/248680/47829720-0d933300-dd46-11e8-9146-30fd19ae0368.png)
![image](https://user-images.githubusercontent.com/248680/47829731-1a178b80-dd46-11e8-8b9f-abe52c3bdc3c.png)

Endor has the best Ewoks

Reply:
... what a dork

Reply:
in the BEST way possible!

Reply:
:nerd_face: